### PR TITLE
Adding officer deaths page

### PR DESCRIPTION
**Pull request by kgilder**

### CODE REVIEW

**Review by nholden — APPROVED** [review]
Awesome refactor! Left a few questions/suggestions. Feel free to merge when ready!

File: components/CovidMap.js [hunk 1]
```diff
@@ -300,11 +300,15 @@ Tick.propTypes = {
 class Map extends React.Component {
   constructor(props) {
     super(props);
+    const { googleSheetsKey, googleSheetsName, allDeathsOnly } = props;
     const today = moment().toDate();
     this.maxSliderDate = today;
-    this.minSliderDate = new Date(2020, 3, 7);
+    this.minSliderDate = new Date(2020, 3, 6);
     this.state = {
+      allDeathsOnly: allDeathsOnly || false,
```
[the remark this answers is not in the document]
Comment (kgilder) [reply components/CovidMap.js:308]: So it does, done!

File: components/CovidMap.js [hunk 2]
```diff
@@ -868,34 +874,42 @@ class Map extends React.Component {
...
-                  name="age"
-                  value="age"
-                  checked={selectedOption === 'age'}
-                  onChange={this.handleOptionChange.bind(this)}
-                  className="form-check-input"
-                />
-                By Age
-              </label>
-            </div>
+            {!allDeathsOnly ? (
```
[the remark this answers is not in the document]
Comment (kgilder) [reply components/CovidMap.js:877]: My end goal is to generate these additional form inputs from an array of 'filters', such as 'age', or 'facility', and then I won't need the ternary operator at all.

File: next.config.js [hunk 3]
```diff
@@ -19,6 +19,9 @@ const nextConfig = {
       '/publications/covid-deaths-in-texas': {
         page: '/publications/covid-deaths-in-texas',
       },
+      // '/publications/covid-officer-deaths': {
```
[the remark this answers is not in the document]
Comment (kgilder) [reply next.config.js:22]: Yes.

File: components/CovidMap.js [hunk 4]
```diff
@@ -868,34 +874,42 @@ class Map extends React.Component {
...
+                    onChange={this.handleOptionChange.bind(this)}
+                    className="form-check-input"
+                  />
+                  Facility Type
+                </label>
+              </div>
+            ) : (
+              <div></div>
+            )}
+            {!allDeathsOnly ? (
```
[the remark this answers is not in the document]
Comment (kgilder) [reply components/CovidMap.js]: Actually I can just remove the need for this now

File: components/CovidMap.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (nholden) [reply components/CovidMap.js]: Gotcha! If we only need the `!allDeathsOnly` case for now (and we don't need the other side of the ternary operator with the empty `<div>`, I'd recommend [Inline If with Logical && Operator](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator).

**Pull request merged**